source/recipes/faq.md

```diff
@@ -2,6 +2,44 @@
 
 ## Nix
 
+### What are flakes?
+
+What is usually referred to as "flakes" is:
+- A policy for managing dependencies between {term}`Nix expression`s.
+- An [experimental feature] in Nix, implementing that policy and supporting functionality.
+
+[experimental feature]: https://nixos.org/manual/nix/stable/contributing/experimental-features.html
+
+Technically, a [flake](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html#description) is a file system tree that contains a {term}`Nix file` named `flake.nix` in its root directory.
```

Comment:
I hate to have to point this out, but it's technically not a Nix file, because not the entire Nix language is supported, see https://github.com/NixOS/nix/issues/4945

Reply:
Thank you. I love how you manage to break things.

Suggestions on how to stick to the truth without neither misleading or confusing people, nor going over board with details?

"a file named `flake.nix`" does not point out the distinction because it leaves open which language it's in (and one may assume the technically wrong thing)

Maybe one could add that bit to where we talk about the schema?

Reply:
Yeah that sounds good. Alternatively how about adding a footnote at "Nix file" and then link to that issue in it.

Reply:
No, I think we should not pollute documentation of stable interfaces with that stuff. Once stabilising flakes can be deemed on the finish line, we can still think about how and where to talk about these things.